index.js

```diff
@@ -32,60 +36,57 @@ module.exports = function (opts) {
 
 		if (validExts.indexOf(path.extname(file.path).toLowerCase()) === -1) {
 			if (opts.verbose) {
-				gutil.log('gulp-imagemin: Skipping unsupported image ' + chalk.blue(file.relative));
+				gutil.log(`gulp-imagemin: Skipping unsupported image ${chalk.blue(file.relative)}`);
 			}
 
 			cb(null, file);
 			return;
 		}
 
-		var imagemin = new Imagemin()
-			.src(file.contents)
-			.use(Imagemin.gifsicle({interlaced: opts.interlaced}))
-			.use(Imagemin.jpegtran({progressive: opts.progressive}))
-			.use(Imagemin.optipng({optimizationLevel: opts.optimizationLevel}))
-			.use(Imagemin.svgo({
-				plugins: opts.svgoPlugins || [],
-				multipass: opts.multipass
-			}));
-
-		if (opts.use) {
-			opts.use.forEach(imagemin.use.bind(imagemin));
-		}
+		const use = plugins || [
+			imageminGifsicle(),
+			imageminMozjpeg(),
+			imageminOptipng({optimizationLevel: 3}),
+			imageminSvgo()
```

Comment:
Yeah, that's weird. I still think it's worth it though. `optipng` also does multiple passes by default.